game-core/build.gradle

```diff
@@ -21,6 +21,7 @@ dependencies {
 
     testImplementation project(':test-common')
     testImplementation "org.sonatype.goodies:goodies-prefs:$sonatypeGoodiesPrefsVersion"
+    testImplementation "com.github.npathai:hamcrest-optional:2.0.0"
```

Comment:
Since this dependency is used in three different subprojects, I'd recommend just applying it to all subprojects in the root _build.gradle_.

If you don't want to do that, then I would recommend at least defining the version number in the root _build.gradle_ to make it harder for them to get out of sync.  (See the previous line for an example.)